src/js/electron/html-dialog/dialog.tsx

```diff
@@ -0,0 +1,52 @@
+import Markdown from "app/core/components/markdown"
+import useKeybinding from "app/core/hooks/use-keybinding"
+import Icon from "app/core/Icon"
+import parseSearchParams from "app/core/utils/parse-search-params"
+import ToolbarButton from "app/toolbar/button"
+import {ipcRenderer} from "electron"
+import React, {useEffect} from "react"
+import {render} from "react-dom"
+import theme from "src/js/style-theme"
+import {ThemeProvider} from "styled-components"
+import {BG} from "./dialog.style"
+import {closeChannel, readyChannel} from "./index"
+
+const ROOT_ID = "root"
+/**
+ * These come from the main process
+ */
+const {title, content, id} = parseSearchParams()
```

Comment:
So we are storing the markdown content in a search param of the url/location?

Reply:
Correct. Since I figured the content of dialog boxes will be short enough to fit into a url. The only other option is to request the content with ipc once the window has rendered.

Reply:
Okay, yeah that seems a bit limiting but I also agree it's unlikely we will have much to say in the dialogs anyway. Will make a mental note of it too.